package.json

```diff
@@ -22,5 +28,45 @@
   "bugs": {
     "url": "https://github.com/esbb48/Kizuna/issues"
   },
-  "homepage": "https://github.com/esbb48/Kizuna#readme"
+  "homepage": "https://github.com/esbb48/Kizuna#readme",
+  "dependencies": {
+    "babel-polyfill": "^6.9.1",
+    "grommet": "^0.6.10",
+    "react": "^15.3.0",
+    "react-dom": "^15.3.0",
+    "react-hot-loader": "^3.0.0-beta.1",
+    "react-redux": "^4.4.5",
+    "react-router": "^2.6.1",
+    "react-router-redux": "^4.0.5",
+    "redux": "^3.5.2",
+    "redux-action": "^1.0.0",
+    "redux-saga": "^0.11.0"
+  },
+  "devDependencies": {
+    "autoprefixer": "^6.3.7",
+    "babel-core": "^6.11.4",
+    "babel-eslint": "^6.1.2",
+    "babel-loader": "^6.2.4",
+    "babel-plugin-transform-object-rest-spread": "^6.8.0",
+    "babel-preset-es2015": "^6.9.0",
+    "babel-preset-react": "^6.11.1",
+    "babel-preset-stage-0": "^6.5.0",
+    "eslint": "^3.2.2",
+    "eslint-config-airbnb": "^10.0.0",
+    "eslint-plugin-import": "^1.12.0",
+    "eslint-plugin-jsx-a11y": "^2.0.1",
+    "eslint-plugin-react": "^6.0.0",
+    "extract-text-webpack-plugin": "^1.0.1",
+    "html-webpack-plugin": "^2.22.0",
+    "precss": "^1.4.0",
+    "stylelint": "^7.1.0",
+    "stylelint-config-standard": "^12.0.0",
+    "url-loader": "^0.5.7",
+    "webpack": "^1.13.1",
+    "webpack-dev-server": "^1.14.1",
+    "webpack-md5-hash": "0.0.5"
+  },
+  "stylelint": {
+    "extends": "stylelint-config-standard"
+  }
 }
```

Comment:
Do you think adding `` engines `` according to [package.json | npm Documentation](package.json | npm Documentation) would be a good idea?